docs/extensions/tutorials/_posts/1970-01-05-Installing3rdPartyPackages.md

```diff
@@ -0,0 +1,323 @@
+
+We begin by initializing an extension and writing in the basic information about it.
+
+```ShellSession
+$ shoutem init swiper-extension
+Enter information about your extension. Press `return` to accept (default) values.
+Title: Swiper Extension
+Version: 0.0.1
+Description: Extension that uses react-native-swiper.
+
+Initializing extension:
+...
+
+Extension initialized!
+```
+
+Let's switch over to the extension directory and add a screen with a shortcut that will show the Swiper.
+
+```ShellSession
+$ cd swiper-extension
+$ shoutem screen add SwiperScreen --shortcut Swiper
+Enter shortcut information:
+Title: Swiper
+
+Screen `SwiperScreen` is created in file `app/screens/SwiperScreen.js`!
+Shortcut `Swiper` is created.
+Shortcut `Swiper` opens `SwiperScreen` screen.
+File `app/extension.js` was modified.
+File `extension.json` was modified.
+```
+
+### Installing the Package
+
+There's two different ways you can do this. One is to manually install the package into the extension using `npm install react-native-swiper --save` in the `app` directory:
+
+```ShellSession
+$ cd app
+$ npm install react-native-swiper --save
+{{ site.example.devName }}.swiper-extension@0.0.1 /absolute/path/swiper-extension/app
+└── react-native-swiper@1.5.4
+```
+
+> #### Note
+> The reason we have to install it into the `app` directory is because the the `app` directory is bundled into the full app along with all the other extensions the app uses.
+
+Doing this will automatically add `react-native-swiper` as a dependency in our `app/package.json` file:
+
+```json{5-7}
+#file: app/package.json
+{
+  "name": "{{ site.example.devName }}.not-swiper",
+  "version": "0.0.1",
+  "description": "Extension that uses react-native-swiper.",
+  "dependencies": {
+    "react-native-swiper": "^1.5.4"
+  }
+}
+```
+
+The other, more preferred way of doing this is to simply add that dependency we see in the `app/package.json` file, which results with `react-native-swiper` being installed when the app is bundled. The reason this is the preferred way of doing it is because the `node_modules` folder generated with our NPM installation is ignored since we utilize the `$ npm pack` command.
+
+### Using the Package
+
+Our extension now has full access to everything `react-native-swiper` has to offer and we can use it just like we would in a normal React Native app, so let's make use of it's simplest example to show how it works. We'll have to edit our `app/screens/SwiperScreen.js` file to use the Swiper by importing the Swiper component, changing the `render();` method to use the Swiper component and making some changes to the `styles` constant so our Swiper component can function properly.
+
+```javascript{12,17-27,38-62}
+#file app/screens/SwiperScreen.js
+import React, {
+  Component
+} from 'react';
+
+import {
+  StyleSheet,
+  Text,
+  View
+} from 'react-native';
+
+import Swiper from 'react-native-swiper';
+
+export default class SwiperScreen extends Component {
+  render() {
```

Comment:
It's explained in the text beforehand. This is the basic swiper example from `react-native-swiper`.